src/dms/core/EdFi.DataManagementService.Core.Tests.Unit/ApiServiceJwtAuthenticationTests.cs

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Reflection;
+using EdFi.DataManagementService.Core.ApiSchema;
+using EdFi.DataManagementService.Core.Configuration;
+using EdFi.DataManagementService.Core.External.Interface;
+using EdFi.DataManagementService.Core.Middleware;
+using EdFi.DataManagementService.Core.Pipeline;
+using EdFi.DataManagementService.Core.ResourceLoadOrder;
+using EdFi.DataManagementService.Core.Security;
+using EdFi.DataManagementService.Core.Validation;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+using Polly;
+
+namespace EdFi.DataManagementService.Core.Tests.Unit;
+
+[TestFixture]
+public class ApiServiceJwtAuthenticationTests
```

Comment:
Not all that interesting of tests, but ensures JWT Auth isn't accidentally disabled in code.